<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>685</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>21696</width>
    <height>13560</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>-7655</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(242,242,242);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(49,67,100);
font:28pt &quot;Sans&quot;;
font:bold;
background-color: none;</string>
    </property>
    <property name="text">
     <string>ARS-Testing</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>831</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>12</y>
       <width>811</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">color: rgb(49,67,100);
font: 11pt &quot;Sans&quot;;
font:bold;
background-color: none;</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">color: rgb(49,67,100);
font: 11pt &quot;Sans&quot;;
font:bold;
background-color: none;</string>
        </property>
        <property name="text">
         <string>ID staff</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">color: rgb(49,67,100);
font: 11pt &quot;Sans&quot;;
font:bold;
background-color: none;</string>
        </property>
        <property name="text">
         <string>Workplace</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(49,67,100);
font: 11pt &quot;Sans&quot;;
font:bold;
background-color: none;</string>
        </property>
        <property name="text">
         <string>Executor</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="mLeNameExecutor">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220,225,226);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="mLeIDExecutor">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220,225,226);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="mLeWorkExecutor">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220,225,226);</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(49,67,100);
font: 11pt &quot;Sans&quot;;
font:bold;
background-color: none;</string>
        </property>
        <property name="text">
         <string>Supervisor</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="mLeNameSupervisor">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220,225,226);</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="mLeIDSupervisor">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220,225,226);</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="mLeWorkSupervisor">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220,225,226);</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">color: rgb(49,67,100);
font: 11pt &quot;Sans&quot;;
font:bold;
background-color: none;</string>
        </property>
        <property name="text">
         <string>Location Test</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="mLeLocalTest">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220,225,226);</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">color: rgb(49,67,100);
font: 11pt &quot;Sans&quot;;
font:bold;
background-color: none;</string>
        </property>
        <property name="text">
         <string>Time Test</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="mLeTimeTest">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220,225,226);</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btn_test">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
border-color: rgb(49, 67, 100);
color: rgb(49,67,100);
background-color: rgb(200, 200, 200); 
font: 87 10pt &quot;Myriad Pro&quot;;
font:bold;
outline:none;
border:none;
}
QPushButton::hover
{
   background-color:rgba(146,165,201);
  	border:none;
  	outline:none;
}</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(&quot;:/Test/images/logo.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMainExit">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>4</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Test/images/button/exit_icon.png</normaloff>:/Test/images/button/exit_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>1121</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="mBtCableTest">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>261</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-color: rgba(220,225,226, 255);
outline: none;
color: rgb(49,67,100);
font: 11pt &quot;Sans&quot;;
font:bold;
border-image: url(&quot;:/Test/images/electric_cable_test.png&quot;);
}
QPushButton::hover
{
   background-color:rgba(176,188,189);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(49,67,100);
font: 14pt &quot;Sans&quot;;
font:bold;
background-color: none;</string>
        </property>
        <property name="text">
         <string>Test Electrical Cable </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="mBtRfTest">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>261</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-color: rgb(220,225,226);
outline: none;
color: rgb(49,67,100);
font: 11pt &quot;Sans&quot;;
font:bold;
border-image: url(&quot;:/Test/images/rf_cable_test.png&quot;);
}
QPushButton::hover
{
   background-color:rgba(176,188,189);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(49,67,100);
font: 14pt &quot;Sans&quot;;
font:bold;
background-color: none;</string>
        </property>
        <property name="text">
         <string>Test RF Cable</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_4">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="mBtModuleTest">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>261</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-color: rgb(220,225,226);
outline: none;
color: rgb(49,67,100);
font: 11pt &quot;Sans&quot;;
font:bold;
border-image: url(&quot;:/Test/images/module_test.png&quot;);
}
QPushButton::hover
{
   background-color:rgba(176,188,189);
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(49,67,100);
font: 14pt &quot;Sans&quot;;
font:bold;
background-color: none;</string>
        </property>
        <property name="text">
         <string>Test Module</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_5">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="mBtMechanicalTest">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>271</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-color: rgb(220,225,226);
outline: none;
color: rgb(49,67,100);
font: 11pt &quot;Sans&quot;;
font:bold;
border-image: url(&quot;:/Test/images/mechanical_test.png&quot;);
}
QPushButton::hover
{
   background-color:rgba(176,188,189);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(49,67,100);
font: 14pt &quot;Sans&quot;;
font:bold;
background-color: none;</string>
        </property>
        <property name="text">
         <string>Test Mechanical</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
